tests/checks/integration/test_process.py

```diff
@@ -321,3 +322,109 @@ def test_check_real_process(self):
+                "btime 1448632481\n"
+            ),
+        })
+
+        config = {
+            'init_config': {
+                'procfs_path': my_procfs
+            },
+            'instances': [{
+                'name': 'moved_procfs',
+                'search_string': [unique_process_name],
+                'exact_match': False,
+                'ignored_denied_access': True,
+                'thresholds': {'warning': [1, 10], 'critical': [1, 100]},
+            }]
+        }
+
+        version = int(psutil.__version__.replace(".", ""))
+        try:
+            def import_mock(name, i_globals={}, i_locals={}, fromlist=[], level=-1, orig_import=__import__):
+                # _psutil_linux and _psutil_posix are the C bindings; use a mock for those
+                if name in ('_psutil_linux', '_psutil_posix') or level >= 1 and ('_psutil_linux' in fromlist or '_psutil_posix' in fromlist):
+                    m = MagicMock()
+                    # the import system will ask us for our own name
+                    m._psutil_linux = m
+                    m._psutil_posix = m
+                    # there's a version safety check in psutil/__init__.py; this skips it
+                    m.version = version
+                    return m
+                return orig_import(name, i_globals, i_locals, fromlist, level)
+
+            orig_open = open
+
+            def open_mock(name, *args):
+                from mock import MagicMock
+
+                # Work around issue addressed here: https://github.com/giampaolo/psutil/pull/715
+                # TODO: Remove open_mock if the patch lands
+                # We can't use patch here because 1) we're reloading psutil, and 2) the problem is happening during the import.
+                # NB: The values generated here are mostly ignored, and will correctly be overwritten once we set PROCFS_PATH
+                if name == '/proc/stat':
+                    handle = MagicMock(spec=file)
+                    handle.write.return_value = None
+                    handle.__enter__.return_value = handle
+                    handle.readline.return_value = 'cpu  13002 0 18504 377363817 1986 2 2960 0 0 0'
+                    return handle
+                return orig_open(name, *args)
+
+            # contextlib.nested is deprecated in favor of with MGR1, MGR2, ... etc, but we have too many mocks to fit on one line and apparently \ line
+            # continuation is not flake8 compliant, even when semantically required (as here). Patch is unlikely to throw errors that are suppressed, so
+            # the main downside of contextlib is avoided.
+            with contextlib.nested(patch('sys.platform', 'linux'),
+                                   patch('socket.AF_PACKET', create=True),
+                                   patch('__builtin__.__import__', side_effect=import_mock),
+                                   patch('__builtin__.open', side_effect=open_mock)):
```

Comment:
I don't have easy access to a Windows machine, but I did develop the test on OS X – what failure are you seeing? It seems to work for me on 10.10 / Python 2.7.10, but I may be doing something differently from you.